tenant/storage_urm.go

```diff
@@ -13,7 +14,57 @@ var (
 	urmByUserIndexBucket = []byte("userresourcemappingsbyuserindexv1")
 )
 
+func getBucket(rt influxdb.ResourceType) ([]byte, error) {
+	var bucket []byte
+	switch rt {
+	case influxdb.UsersResourceType:
+		bucket = userBucket
+	case influxdb.OrgsResourceType:
+		bucket = organizationBucket
+	case influxdb.BucketsResourceType:
+		bucket = bucketBucket
+	default:
```

Comment:
This kind of switch has to be avoided. Once we add more systems in that use the tenant urm system it would be tempting to add non tenant types to this and that would enforce that all systems exist in the same database. We know that will not be the case in the future.

Reply:
Yeah,  I think you are right 👍

Reply:
See below for more